frigate/events/audio.py

```diff
@@ -283,10 +305,40 @@ def log_and_restart() -> None:
             self.logger.error(f"Error reading audio data from ffmpeg process: {e}")
             log_and_restart()
 
+    def _update_enabled_state(self) -> bool:
+        """Fetch the latest config and update enabled state."""
+        _, config_data = self.enabled_subscriber.check_for_update()
+        if config_data:
+            enabled = config_data.enabled
```

Comment:
See the changes I made to the other instances, we should just be changing the config enabled value directly and return that

Reply:
I took this from the latest output.py, did you mean to change something there that didn't get changed?